recursion/towersOfHanoi/TowersOfHanoi.go

```diff
@@ -0,0 +1,5 @@
+package TowersOfHanoi
+
+type TOHCalculator interface {
```

Comment:
I like that you are using interface here. Good job 👍

Reply:
thank you